inputs/range.vue

```diff
@@ -230,6 +230,10 @@
           el.noUiSlider.on(key, this.schema.events[key]);
         });
       }
+
+      if (this.disabled) {
+        el.setAttribute('disabled', this.disabled);
+      }
```

Comment:
Can't we add it directly to this line: https://github.com/clay/clay-kiln/blob/2d5759f9ca024859363d3562052a505ccb8f5475/inputs/range.vue#L40 as `:disabled="disabled"` to keep consistency with the others?

Reply:
The attribute is being set on the slider created in the `mounted` hook, not the actual element